### ISSUE TITLE
Not possible ingest tiles in some source projections.

### ISSUE THREAD

**Issue by pomadchin** [locationtech/geotrellis#1415]
Ingest issue can be repeated by using this projection:

``` bash
+proj=cea +lon_0=-117.333333333333 +lat_ts=33.75 +x_0=0 +y_0=0 +datum=NAD27 +units=m +no_defs
```

Or even projection with EPSG code (we have it in our proj4 tests):

``` bash
# NSIDC EASE-Grid Global
<3410> +proj=cea +lon_0=0 +lat_ts=30 +x_0=0 +y_0=0 +a=6371228 +b=6371228 +units=m +no_defs  <>
```

Probably there are more projections, we have to check out possible bugs more accurate.

**Comment by pomadchin** [comment]
Mentioning there @dwins and @lossyrob

**Comment by pomadchin** [comment]
* https://github.com/biggis-project/biggis-landuse/issues/2

**Comment by metasim** [comment]
@pomadchin Might this potentially be addressed by #2169? Or is it more like #2345? Any details on what the specific problem is, and if there's a test case already?
